### DISCUSSION TITLE
Does a github action for gocd exist?

### DISCUSSION THREAD

**Discussion by fire** [gocd/gocd#11967] [Q&A]
Does a github action for gocd exist? Something that's entirely on github cloud services.

**Comment by chadlwilson** [comment]
What would such a Github action do?

**Reply by fire** [reply]
Start a docker instance and runs gocd.

**Reply by fire** [reply]
Like https://github.com/marketplace/actions/argo-cd-action

**Reply by chadlwilson** [reply]
As far as I know, that action simply runs the argocd CLI tool. It doesn;t run an entire argocd server, since the server would have to be separately running within a Kubernetes cluster as a kubernetes-native tool.

An equivalent to that for GoCD would be an action that uses an existing GoCD CLI tool or other mechanism to invoke the GoCD API to trigger builds or configure a GoCD server running somewhere else.

In any case, I'm not aware of any GitHub Actions related to GoCD besides https://github.com/marketplace/actions/gocd-mergeable
